yetanotherco/aligned_layer
# fix (batcher): ws reconnections

- Forked ethers to https://github.com/yetanotherco/ethers-rs.git
 - Batcher now keeps trying to reconnect every one second if reconnection fails
 
 # To test

- Start anvil and batcher
- Kill anvil
- Batcher should try to reconnect once per second, logging a warning when failing
- Start anvil
- Batcher should reconnect
